css-grid-1/Overview.bs

```diff
@@ -4902,6 +4858,10 @@ Changes since the <a href="https://www.w3.org/TR/2020/CRD-css-grid-1-20201218/">
 				instead use the <a>grid area</a> determined in [[#abspos-items]].</del>
 			</blockquote>
 
+		<li id="change-2020-fr-unit">
+			Moved the definition of <<fraction>> to [[css-values-4#fractions]],
```

Comment:
I think it should be *replaced `<flex>` with `<fraction>` and moved its definition to ...*.

Reply:
Yep. Changed.